Server/htdocs/AppController/commands_RSM/utilities/RStools.php

```diff
@@ -539,4 +539,97 @@ function is_base64($s){
 
     return true;
 }
-?>
+
+// Set the Authorization token readed on the header and puts it in the $GLOBALS variable 
+function setAuthorizationTokenOnGlobals() {
+    //We need this variable to exists in order to make RSdatabase work propertly.
+    $GLOBALS['RS_POST']['RStoken'] = getallheaders()["Authorization"];
+    if (strpos($GLOBALS['RS_POST']['RStoken'], "Basic ") !== false) {
+        $GLOBALS['RS_POST']['RStoken'] = str_replace("Basic ", "", $GLOBALS['RS_POST']['RStoken']);
+        $GLOBALS['RS_POST']['RStoken'] =  base64_decode($GLOBALS['RS_POST']['RStoken']);
+        $GLOBALS['RS_POST']['RStoken'] = str_replace("Authorization:", "", $GLOBALS['RS_POST']['RStoken']);
+        $GLOBALS['RS_POST']['RStoken'] = str_replace(";", "", $GLOBALS['RS_POST']['RStoken']);
+    }
+}
+
+// Replace incorrect values related to UTF-8
+function replaceUtf8Characters($propertyValue) {
+    $parsedPropertyValue = str_replace("&amp;", "&", htmlentities($propertyValue, ENT_COMPAT, "UTF-8"));
+    return str_replace("'", "&#39;", $parsedPropertyValue);
```

Comment:
Instead of declaring a new variable, that doubles the RAM consumption, reuse the propertyValue variable

Reply:
This is very important if propertyValue is a file or an image